stdlib/public/core/Integers.swift.gyb

```diff
@@ -3161,16 +3163,14 @@ ${assignmentOperatorComment('%', True)}
   @_inlineable // FIXME(sil-serialize-all)
   @_transparent
   public static func %=(_ lhs: inout ${Self}, _ rhs: ${Self}) {
-    // No LLVM primitives for checking overflow of division
-    // operations, so we check manually.
+    // No LLVM primitives for checking overflow of division operations, so we
+    // check manually.
     if _slowPath(rhs == (0 as ${Self})) {
-      _preconditionFailure("Remainder of division by zero")
+      _preconditionFailure("Division by zero")
     }
 %     if signed:
-    if _slowPath(
-      ${'lhs == %s.min && rhs == (-1 as %s)' % (Self, Self)}
-    ) {
-      _preconditionFailure("Overflow in remainder of division")
+    if _slowPath(${'lhs == %s.min && rhs == (-1 as %s)' % (Self, Self)}) {
+      _preconditionFailure("Division results in an overflow")
```

Comment:
Same here. With the new messages it is impossible to tell which one of the operations failed in the expression like `a / b % c`.